chain-test/src/test/scala/org/bitcoins/chain/util/ChainUnitTest.scala

```diff
@@ -94,4 +99,59 @@ trait ChainUnitTest
 
     dropTableP.future.flatMap(_ => testExecutionF)
   }
+
+  /** Represents a bitcoind instance paired with a chain handler via zmq */
+  case class BitcoindChainHandler(
+      bitcoindRpc: BitcoindRpcClient,
+      chainHandler: ChainHandler)
+
+  def bitcoindZmqChainHandler(test: BitcoindChainHandler => Future[Assertion])(
+      implicit system: ActorSystem): Future[Assertion] = {
+    val instance = BitcoindRpcTestUtil.instance()
+    val bitcoindF = {
```

Comment:
could do 
```scala
val bitcoindF =  BitcoindRpcTestUtil.startedBitcoindRpcClient()
```

Reply:
This is not the case, as we need to construct from the gensis block. `startedBitcoindRpcClient()` generates 100 blocks for us before returning so that the client has money in the wallet.